src/components/FilterBarInline/FilterBarInline.tsx

```diff
@@ -0,0 +1,142 @@
+import { chakra, ChakraComponent, Stack } from "@chakra-ui/react";
+import React, { forwardRef } from "react";
+
+import Button from "../Button/Button";
+import ButtonGroup from "../ButtonGroup/ButtonGroup";
+import { LayoutTypes } from "../../helpers/types";
+import { MultiSelectWidths, SelectedItems } from "../MultiSelect/MultiSelect";
```

Comment:
I think `SelectedItems` is a bit confusing here and in the args table. What do you think about renaming this `SelectedItems as MultiSelectItems`? Then it's clear that the `FilterBarItemsType` accepts data from the `MultiSelect` component. I think that might still be confusing and don't have a good answer for this.